app/vmui/packages/vmui/src/components/ExploreAlerts/NotifierHeader/index.tsx

```diff
@@ -0,0 +1,30 @@
+import React, { FC } from "preact/compat";
+import "./style.scss";
+import useDeviceDetect from "../../../hooks/useDeviceDetect";
+import { Notifier } from "../../../types";
+
+interface NotifierHeaderControlsProps {
+  notifier: Notifier
+}
+
+const NotifierHeaderHeader: FC<NotifierHeaderControlsProps> = ({
+  notifier,
+}) => {
+  const { isMobile } = useDeviceDetect();
+
+  if (isMobile) {
+    return (
+      <div className="vm-explore-alerts-notifier-header vm-explore-alerts-notifier-header_mobile">
+        <div className="vm-explore-alerts-notifier-header__name">{notifier.kind}</div>
+      </div>
+    );
+  }
+
+  return (
+    <div className="vm-explore-alerts-notifier-header">
+      <div className="vm-explore-alerts-notifier-header__name">{notifier.kind}</div>
+    </div>
+  );
+};
```

Comment:
Maybe will be better to join mobile and desktop version into one
```suggestion
  return (
    <div className={classNames({
        "vm-explore-alerts-notifier-header": true, 
        "vm-explore-alerts-notifier-header_mobile": isMobile
        })}
     >
      <div className="vm-explore-alerts-notifier-header__name">{notifier.kind}</div>
    </div>
  );
};
```